invisible_cities/io/dst_io.py

```diff
@@ -109,7 +109,7 @@ def df_writer(h5out              : tb.file.File ,
 
     data_types = table.dtype
     if len(arr) == 0:
-        warnings.warn(f'dataframe is empty', UserWarning)
+        warnings.warn('dataframe is empty', UserWarning)
```

Comment:
I know that this is not strictly related to this PR but I wonder if we really need this warnings. For example, in pandas `to_hdf` no warning is raised if dataframe is empty, so maybe we should just keep the same behavior, ie removing the warning completely.  Especially since it is almost never guaranteed to have non-empty df in our flows...

Reply:
Anybody care to share an opinion on this?

Does anyone ever notice or appreciate this warning appearing? Can anyone remember an instance when this warning helped them? If not, it should probably go.

Reply:
Marija and I discussed it and we decided to remove it in the final version of this PR. That's why it shows as outdated above.

Reply:
I think that it shows as outdated, because you have rebased, and the commit id has changed. In my latest checkout of the branch, I can still see the warning being raised (strangely enough, with the superfluous 'f' which was removed here!)

Reply:
Strange. It's changed in the first commit of the current history and I can't find anywhere in the other commits where it could have been accidentally reintroduced.

Reply:
Apologies. I failed to notice that a Git LFS error prevented me from looking at the right branch. I confirm that it's gone from the current version.